sopel/irc.py

```diff
@@ -148,6 +148,16 @@ def write(self, args, text=None):
             self.send(temp.encode('utf-8'))
         finally:
             self.writing_lock.release()
+        # Simulate echo-message
+        if 'echo-message' not in self.enabled_capabilities:
```

Comment:
As [@maxpowa mentioned](https://github.com/sopel-irc/sopel/pull/1072#issuecomment-221168631) in the main PR comments, "As it is right now, it will echo every type of message, but the echo message spec only includes `NOTICE` and `PRIVMSG`." Therefore, this block should only fire if the message is one of those types.